lib/cloudinary/utils.rb

```diff
@@ -996,7 +997,22 @@ def self.generate_auth_token(options)
     Cloudinary::AuthToken.generate options
 
   end
-  
+
+  # Creates and returns a URL that when invoked creates an archive of a folder.
+  #
+  # @param [Object] folder_path Full path (from the root) of the folder to download.
+  # @param [Hash] options       Additional options.
+  #
+  # @return [String]
+  def self.download_folder(folder_path, options = {})
```

Comment:
Please move this function closer to `download_archive_url`, it's just an alias with prepopulated `prefix` and `resource_type`.